packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/document/DocumentAsset.java

```diff
@@ -53,6 +54,7 @@ public class DocumentAsset extends TerariumAsset {
 	@TSOptional
 	@Type(JsonType.class)
 	@Column(columnDefinition = "json")
+	@Deprecated
```

Comment:
Why not removing them?

Reply:
The removal/cleanup changes too many things as to distract away from the task, these deprecated fields are still in heavy rotation - just that we don't use the processing results (as far as I am aware). Also need to do database migration related matters.